library/extractor/src/main/java/com/google/android/exoplayer2/extractor/mp4/PsshAtomUtil.java

```diff
@@ -188,7 +208,15 @@ private static PsshAtom parsePsshAtom(byte[] atom) {
     }
     byte[] data = new byte[dataSize];
     atomData.readBytes(data, 0, dataSize);
-    return new PsshAtom(uuid, atomVersion, data);
+
+    if (atomVersion == 0) {
+      ParsableByteArray schemeData = new ParsableByteArray(data);
+      if (schemeData.limit() >= 4 + 16 /* UUID */) {
+        schemeData.setPosition(4);
+        keyIds = new UUID[] {new UUID(schemeData.readLong(), schemeData.readLong())};
+      }
+    }
+    return new PsshAtom(uuid, atomVersion, data).withKeyIds(keyIds);
```

Comment:
The return seems to have changed from ExoPlayer's return. Do we need this change?